### PR TITLE
docs: update costs with public ipv4 costs

### PR DESCRIPTION
**Pull request by dengmingtong**
----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

## Summary

add public ip cost into docs

## Implementation highlights
mkdocs serve -f mkdocs.zh.yml
mkdocs serve -f mkdocs.en.yml

Screenshot 2024-04-13 at 18 09 30

Screenshot 2024-04-13 at 18 09 46

Screenshot 2024-04-13 at 18 10 49
Screenshot 2024-04-13 at 18 10 58

## Test checklist

- [ ] add new test cases
- [ ] all code changes are covered by unit tests
- [ ] end-to-end tests
  - [ ] deploy web console with CloudFront + S3 + API gateway
  - [ ] deploy web console within VPC
  - [ ] deploy ingestion server
    - [ ] with MSK sink
    - [ ] with KDS sink
    - [ ] with S3 sink
  - [ ] deploy data processing
  - [ ] deploy data modeling
    - [ ] new Redshift Serverless
    - [ ] provisioned Redshift
    - [ ] Athena
  - [ ] deploy with reporting
  - [ ] streaming ingestion
    - [ ] with Redshift Serverless
    - [ ] with provisioned Redshift

## Is it a breaking change

- [ ] add parameters without default value in stack
- [ ] introduce new service permission in stack
- [ ] introduce new top level stack module

## Miscellaneous

- [ ] introduce new symbol link source file(s) to be shared among infra code, web console frontend, and web console backend

### CODE REVIEW

File: docs/en/plan-deployment/cost.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ The following are cost estimations for monthly data volumes of 10/100/1000 RPS (
 
 Ingestion module includes the following cost components:
 
-- Application load balancer
+- Application load balancer and Public IP
```
Comment (zxkane) [inline docs/en/plan-deployment/cost.md:33]: -> Public IPv4 Address

File: docs/en/plan-deployment/cost.md [hunk 2]
```diff
@@ -41,21 +41,22 @@ Key assumptions include:
...
-|                         | $18    |  $122 |  None              |             |  $3    |      $143   |
-|100RPS (490GB/month)          |  $43  |  $122  |  Kinesis(On-demand)              |      $115       |  $4   |     $284 |
-|                         | $43    |   $122 |  Kinesis (Provisioned 2 shard)   |      $26       | $4    |     $195  |
-|           |   $43  |  $122  |   MSK (m5.large * 2, connector MCU * 1)              |      $417       |  $4   |     $586
-|           |   $43  |  $122 |      None              |             |  $4    |     $169
-|1000RPS (4900GB/month)          |   $252  |   $122 |      Kinesis(On-demand)              |      $1051       |  $14   |    $1439 |
-|                         |  $252   |  $122  |  Kinesis (Provisioned 10 shard)   |    $180         |   $14  |     $568  |
-|           |  $252   | $122  |      MSK (m5.large * 2, connector MCU * 2~3)              |      $590       |  $14  |     $978
-|           |  $252   | $122   |      None              |            |  $14   |     $388 
+| 10RPS (49GB/month)             |  $21.6  |  $122 |  Kinesis (On-Demand) |    $38       |   $3  |     $184.6  |
```
Comment (zxkane) [inline docs/en/plan-deployment/cost.md:48]: Each public IPv4 address charges $0.005 per hour.

Three IPv4 addresses will cost 3.6 (30 days) * 3 = 10.8 per month.

File: docs/en/plan-deployment/cost.md (hunk 1 above)
Comment (zxkane) [reply docs/en/plan-deployment/cost.md:33]: For reference, https://aws.amazon.com/blogs/aws/new-aws-public-ipv4-address-charge-public-ip-insights/

**Review by zxkane — CHANGES_REQUESTED** [review]
- [ ] update for v1.1.x
- [ ] update AGA costs too

File: docs/en/plan-deployment/cost.md (hunk 1 above)
Comment (dengmingtong) [reply docs/en/plan-deployment/cost.md]: Refactored

File: docs/en/plan-deployment/cost.md (hunk 2 above)
Comment (dengmingtong) [reply docs/en/plan-deployment/cost.md]: Fixed

**Review by zxkane — APPROVED** [review]

**Review by zxkane — CHANGES_REQUESTED** [review]
It's an update for latest doc, so the target branch should be 1.1.x.

**Pull request closed without merging**